src/integration-tests/bash/run.sh

```diff
@@ -3015,11 +3036,11 @@ function test_suite {
     kubectl create namespace test1 2>&1 | sed 's/^/+/g' 
     kubectl create namespace test2 2>&1 | sed 's/^/+/g' 
 
-    if ! [ "$USE_HELM" = "true" ]; then
-      # do not create ooperator namespace when using helm charts
-      kubectl create namespace weblogic-operator-1 2>&1 | sed 's/^/+/g' 
-      kubectl create namespace weblogic-operator-2 2>&1 | sed 's/^/+/g' 
-    fi
+    # do not create operator namespace when using helm charts
```

Comment:
Should we remove this comment since we need to create operator namespace now?

Reply:
yes